firebase.json

```diff
@@ -11,6 +11,11 @@
   	  "source" : "/widgets/",
   	  "destination" : "/basic-widgets/",
   	  "type" : 301
-  	}
+  	},
+    {
+      "source" : "/tutorial/",
+      "destination" : "/introducing-flutters-widget-framework/",
```

Comment:
Trying to optimize for SEO, I'm assuming people are cutting and pasting, or searching and clicking, and not manually typing these paths.